src/public/member/editProduct.php

```diff
@@ -0,0 +1,94 @@
+<?php
+if (!isset($_SESSION['user'])) {
+    header('Location: /');
+    exit();
+}
+
+if (isset($_GET['id'])) {
+    $productId = $_GET['id'];
+    $sellerId = $_SESSION['user']['id'];
+
+
+    require_once $_SERVER['DOCUMENT_ROOT'] . '/config.php';
+    require_once DATABASE . '/connect.php';
+
+    $sql = "SELECT p.*, c.name AS category_name FROM products p 
+            LEFT JOIN product_categories c ON p.categoryid = c.id 
+            WHERE p.id = ? AND p.userid = ?";
+
+    $stmt = mysqli_prepare($connection, $sql);
+
+    if ($stmt) {
+        mysqli_stmt_bind_param($stmt, "ii", $productId, $sellerId);
+        mysqli_stmt_execute($stmt);
+        $result = mysqli_stmt_get_result($stmt);
+
+        if (mysqli_num_rows($result) >= 1) {
+            $row = mysqli_fetch_assoc($result);
+            $productName = $row['name'];
+            $productDescription = $row['description'];
+            $productPrice = $row['price'];
+            $productImageUrl = $row['imageUrl'];
+            $productCategory = $row['category_name'];
+    
+          
+        } else {
+            echo "Debug: Product not found.";
+        }
+
+        mysqli_stmt_close($stmt);
+    } else {
+        die("Prepared statement error: " . mysqli_error($connection));
+    }
+} else {
+    echo "Debug: Product ID is missing.";
+}
+
+if (isset($_SESSION['update_successfully'])) {
+    echo '
+      <div class="alert alert-success">
+          <svg xmlns="http://www.w3.org/2000/svg" class="stroke-current shrink-0 h-6 w-6" fill="none" viewBox="0 0 24 24"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M9 12l2 2 4-4m6 2a9 9 0 11-18 0 9 9 0 0118 0z" /></svg>
+          <span>' . $_SESSION['update_successfully'] . '</span>
+      </div>';
+}
+if (isset($_SESSION['no_changes1'])) {
+    echo '
+      <div class="alert alert-info">
+          <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" class="stroke-current shrink-0 w-6 h-6"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M13 16h-1v-4h-1m1-4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"></path></svg>
+          <span>' . $_SESSION['no_changes'] .
+          '</span>
+      </div>';
+}
+?>
+
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Edit Product</title>
+
+</head>
+<body>
```

Comment:
You don't need to put html, head, title, body tags in all files, these files get included in the main index where they are already surrounded by those tags.